File: Percentage_of_Letter_in_String.py
s = "foobar"
letter = "o"


def percentageLetter(s,letter):
    count = 0
    for i in s:
        if i == letter:
            count+=1


    if count > 0:
        return int(count / len(s) *100)
    return count


print(percentageLetter(s,letter))

File: Percentage_of_Letter_in_String/Percentage_of_Letter_in_String.go
package main

import (
	"fmt"
)

func main() {
	s := "foobar"
	letter := "o"
	fmt.Println(percentageLetter(s, letter))
}

func percentageLetter(s string, letter string) int {
	var count int = 0
	for i := 0; i < len(s); i++ {
		switch string(s[i]) {
		case letter:
			count++

		}
	}

	return int(float64(count) / float64(len(s)) * 100)

}
